gitlint-core/gitlint/tests/contrib/rules/test_authors_commit.py

```diff
@@ -0,0 +1,88 @@
+from collections import namedtuple
+from unittest.mock import Mock, patch
+
+import pytest
+
+from gitlint.contrib.rules.authors_commit import Authors
+
+
+Author = namedtuple("Author", "name, email")
+GitCtx = namedtuple("GitCtx", "repository_path")
+MockFile = namedtuple("MockFile", "name")
+
+
+@pytest.fixture(name="git_ctx")
+def gen_git_ctx(tmpdir):
+    authors_file = tmpdir / "AUTHORS"
+    authors_file.write_text("John Doe <john.doe@mail.com>\nBob Smith <bob.smith@mail.com>", "utf-8")
+    return GitCtx(tmpdir)
+
+
+AUTHOR_1 = Author("John Doe", "john.doe@mail.com")
+AUTHOR_2 = Author("Bob Smith", "bob.smith@mail.com")
+
+
+@pytest.mark.parametrize("author", [AUTHOR_1, AUTHOR_2])
+def test_authors_succeeds(author, git_ctx):
+    assert not validate_authors_rule(author.name, author.email, git_ctx)
```

Comment:
We should match existing test styling. While correct, we should stay consistent in the project and not use pytest style tests.